### DISCUSSION TITLE
Customized Theme

### DISCUSSION THREAD

**Discussion by hughsaffar** [n1crack/vuefinder#65] [Q&A]
I really liked working with this package. The only drawback for me is that the styles are embedded into the components, and there is no easy way to override them 😭 I'm just curious if anyone has been able to customize the theme to match their application, or if there are any plans to add a feature that allows you to use your own theme?

**Comment by n1crack** [comment]
You are right. At least there should be some css variables to change colors, border radius, fonts etc. 

There are many things in my mind, and applying them as soon as possible in my limited time.

**Reply by hughsaffar** [reply]
@n1crack thanks for your prompt reply. Would it be okay if I create a pull request? My plan is to add a class to each section to allow the user to define the styles for each class. Users can then choose to use vanilla CSS or utilize TailwindCSS, which you are currently using.

**Reply by n1crack** [reply]
Well, I'd be happy if you do. But when doing that, I want to consider naming each class and css variables carefully without breaking current theme.

**Reply by hughsaffar** [reply]
I will take a very non-destructive approach for the sake of backward compatibility.

**Comment by n1crack** [comment]
One step further, we will have a multi-theme version very soon.

https://github.com/user-attachments/assets/05ef44c2-9b74-4db4-99ed-a4fe8f935e8a
